docs/components/receive.md

```diff
@@ -12,6 +12,20 @@ For more information please check out [initial design proposal](../proposals-don
 
 > NOTE: As the block producer it's important to set correct "external labels" that will identify data block across Thanos clusters. See [external labels](../storage.md#external-labels) docs for details.
 
+## Series distribution algorithms
+
+The Receive component currently supports two algorithms for distributing timeseries across Receive nodes and can be set using the `receive.hashrings-algorithm` flag.
+
+### Hashmod (discouraged)
+
+This algorithm uses a `hashmod` function over all labels to decide which receiver is responsible for a given timeseries. This is the default algorithm due to historical reasons. However, its usage for new Receive installations is discouraged since adding new Receiver nodes leads to series churn and memory usage spikes.
+
+### Ketama (recommended)
+
+The Ketama algorithm is a consistent hashing scheme which enables stable scaling of Receivers without the drawbacks of the `hashmod` algorithm. This is the recommended algorithm for all new installations.
+
+If you are using the `hashmod` algorithm and wish to migrate to `ketama`, the simples and safest way would be to set up a new pool receivers with `ketama` hashrings and start remote-writing to them. Provided you are on the latest Thanos version, old receivers will flush their TSDBs after the configured retention period and will upload blocks to object storage. Once you have verified that is done, decommission the old receivers.
```

Comment:
@PhilipGough @douglascamata any suggestions on the migration process?

Reply:
I think you have covered it here. Duplicate the writes two both sets of receivers. We used a proxy to mirror traffic. There might be some temporary impact on the read path after the fact since it can fetch more data and you may want to configure the read path differently during the migration (to read both sets of receivers) if required.